docs/concepts/mixer.md

```diff
@@ -7,40 +7,210 @@ title: Mixer
 type: markdown
 ---
 {% capture overview %}
-The page explains the role and composition of the Istio mixer.
+The page explains the Istio mixer's role and general architecture.
 {% endcapture %}
 
 {% capture body %}
-## General usage
+## Background
 
 The mixer provides the control-plane abstractions necessary for most real-world multi-tenant services.
 The proxy delegates policy decisions to the mixer and dispatches its telemetry data to the mixer, which
 proceeds to repackage and redirect the data towards configured backends.
 
-Services within the Istio mesh can also directly integrate with the mixer. For example, services may wish to provide rich telemetry for particular operations
-beyond what the proxy automatically collects. Or services may use the mixer for resource-oriented quota management. Services that leverage the mixer in this 
-way are abstracted from environment-specific control plane details, greatly easing the process of hosting the code in
-different environments (different clouds & on-prem)
+Services within the Istio mesh can also directly integrate with the mixer. For example, services may wish to provide rich telemetry
+for particular operations beyond what the proxy automatically collects. Or services may use the mixer for resource-oriented quota
+management. Services that leverage the mixer in this way are abstracted from environment-specific control plane details, greatly
+easing the process of  hosting the code in different environments (different clouds & on-prem)
+
+<img style="display:block;width:60%;margin:auto;" src="./mixer/ingress.svg" alt="Flow of inbound traffic to a service." />
+<p style="text-align:center;">Ingress Traffic</p><br/>
+
+<img style="display:block;width:60%;margin:auto;" src="./mixer/egress.svg" alt="Flow of outbound traffic to a service." /> 
+<p style="text-align:center;">Egress Traffic</p>
 
 The mixer provides three core features:
 
-**Precondition Checking**. Enables callers to verify a number of preconditions before responding to an incoming request from a service consumer. 
+- **Precondition Checking**. Enables callers to verify a number of preconditions before responding to an incoming request from a service consumer. 
```

Comment:
In the preceding text, we refer to this as "policy decisions". We probably should be consistent.